app/javascript/components/utils/ApplicationReview.jsx

```diff
@@ -0,0 +1,184 @@
+import React from "react";
+import axios from 'axios';
+import PropTypes from "prop-types";
+import linkedin from "images/linked-in.png";
+
+class ApplicationReview extends React.Component {
+    constructor(props) {
+        super(props);
+        axios.defaults.headers.common = {
+            "X-Requested-With": "XMLHttpRequest",
+            "X-CSRF-TOKEN": document
+              .querySelector('meta[name="csrf-token"]')
+              .getAttribute("content")
+          };
+        this.handleDeny = this.handleDeny.bind(this);
+        this.handleInterview = this.handleInterview.bind(this);
+        this.handleAccept = this.handleAccept.bind(this);
+        this.handleVolunteerClick = this.handleVolunteerClick.bind(this);
+        this.displayButtons = this.displayButtons.bind(this);
+        this.goBack = this.goBack.bind(this);
+    }
+
+    handleDeny = e => {
+    e.preventDefault();
+    axios
+        .post(`/api/applications/${this.props.application.id}/decide`, {
+        decision: 'denied'
+        })
+        .then(function(response) {
+        window.location.reload();
+        })
+        .catch(function(error) {
+        console.log(error);
```

Comment:
Use flash error messages here. See my error message handler PR here for instructions on how to use it: https://github.com/calblueprint/rising-tides/pull/91

You can just do the error catching function here.

You would do something like this:
```
.catch(function(error) {
  this.flash_message.flashError(
    error.response.data.message
  );
});